vmdb/app/controllers/ems_common.rb

```diff
@@ -583,7 +583,7 @@ def set_form_vars
     @edit[:new][:hostname] = @ems.hostname
     @edit[:new][:ipaddress] = @ems.ipaddress
     @edit[:new][:emstype] = @ems.emstype
-    @edit[:amazon_regions] = get_amazon_regions if @ems.emstype == "ec2"
+    @edit[:amazon_regions] = get_amazon_regions if @ems.kind_of?(EmsAmazon)
```

Comment:
I started doing that and realized the UI code is so amazon specific, with the `:amazon_regions` and `get_amazon_regions`, that it looked stupid, so I left it like this.